### DISCUSSION TITLE
Difference between snapshot taken using `etcdctl snapshot save` and  copying the member/snap/db file from an etcd data directory.

### DISCUSSION THREAD

**Discussion by haoweig** [etcd-io/etcd#14801] [Q&A]
I am trying to configure a setup to allow my system to recover after losing quorum permanently. https://etcd.io/docs/v3.5/op-guide/recovery/ provides a detailed step-by-step guide on how to do it but I'm wondering what is the difference between the 2 methods used to get a snapshot of etcd? Will 1 be more updated than the other or are they actually the same thing?

**Comment by lavacat** [comment]
If you are using live instance, then you should use `etcdctl snapshot save`, because implementation will lock db while doing copy.
Also `snapshot save` adds [hash](https://github.com/etcd-io/etcd/blob/3cf2e79e16a7a0e08de0ac168c829b4df20ae644/server/etcdserver/api/v3rpc/maintenance.go#L120) that will be verified on restore.

**Reply by haoweig** [reply]
If I were to use `etcdctl snapshot save`, would the snapshot produced be different if i do it on different nodes on the same cluster?
